src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir.ttc);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+:root {
+  --text-color: #3d4e61;
+  --border-color: #e1e7ed;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  font-weight: 300;
+}
+
+.searchBar {
+  display: block;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 900;
+  color: var(--text-color);
+  width: 100%;
+  height: 70px;
+  background-color: #fff;
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  padding-left: 62px;
+  outline: none;
+  padding-bottom: 3px;
+}
+
+.searchBar::placeholder {
+  font-weight: normal;
+  color: var(--text-color);
+  font-family: "Avenir", sans-serif;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  outline: none;
+}
+
+.searchBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid var(--border-color);
+  border-radius: 3px;
+  outline: none;
+  font-weight: 900;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.searchBar:focus::placeholder {
+  text-shadow: none;
+}
+
+.container {
+  position: relative;
+  margin: 20px 8px 0;
+}
+
+.container.small {
+  height: 42px;
+}
+
+.container.small .searchBar {
```

Comment:
you did nice work with icon styling, try to apply the same approach here - create search and search--small classes. You can refer to checklist for further details 
![Screenshot 2023-03-21 at 17 42 04](https://user-images.githubusercontent.com/6526149/229152337-f7682442-78f0-43eb-9d08-9d49d2cef25c.png)

Reply:
done